actions/shared_create_releases/action.yaml

```diff
@@ -0,0 +1,45 @@
+name: Create releases
+description: Create releases and build changelogs
+
+inputs:
+  is_release:
+    description: True if a release should be created
+    default: false
+  version_number_input:
+    description: If set, the version number will not be incremented and the given number will be used.
+    default: ''
+  tags:
+    description: The docker image tags
+    default: ''
+  image_tag:
+    description: The image tag for the new version
+    default: ''
+  github_token:
+    description: The github token to login to Docker container
+    required: true
+
+runs:
+  using: "composite" 
+
+  steps:
+    - name: Build Changelog
+      id: github_release
+      if: ${{ inputs.is_release == 'true' }}
+      uses: mikepenz/release-changelog-builder-action@v3
+      with:
+        toTag: v${{ inputs.version_number_input }}
+        configurationJson: |
+          {
+            "template": "#{{CHANGELOG}}\n## Installation\n```\ndocker pull ${{ inputs.tags }}\n```\n\n## Image reference to deploy: \n```\n${{ inputs.image_tag }}\n```\n\n## Changelog\n#{{UNCATEGORIZED}}",
```

Comment:
This text is very docker specific - compare it in the maven pipeline. Suggest we either have the 2 different versions of the text here and an if statement to select between them, or we pass it in